### DISCUSSION TITLE
fail2ban or other rate limiting for docker image?

### DISCUSSION THREAD

**Discussion by cpath-star** [requarks/wiki#6537] [Help / Questions] [answered]
I'm trying to put some security in place for a private wiki. I'd like to have some sort of rate limiting set on failed logins. I'm running the latest docker image but can't seem to find a way to do this. The logs don't appear to indicate failed login attempts, and I don't see anything in the documentation about how to configure something like this or if it's even necessary. Any ideas?

**Comment by NGPixel** [comment, marked as answer]
Wiki.js already has a built-in limiter on failed login attempts.

**Reply by cpath-star** [reply]
terrific, thanks!
